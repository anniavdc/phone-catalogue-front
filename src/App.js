import React from 'react';

// libraries
import {
  BrowserRouter as Router,
  Route,
  Switch
} from 'react-router-dom';

// components
import Main from './pages/Main';
import NotFound from './pages/NotFound';

function App() {
  return (
    <Router>
      <Switch>
        <Route path="/app" component={Main} />
        <Route exact path="/" component={Main} />
        <Route path="*" component={NotFound} />
      </Switch>
    </Router>
  );
}

export default App;
