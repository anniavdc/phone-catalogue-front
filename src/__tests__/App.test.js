import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

test('render', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
