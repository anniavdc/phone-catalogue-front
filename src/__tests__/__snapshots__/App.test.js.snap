// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div>
  <div
    class="main-page"
  >
    <header>
      Header
    </header>
    <section
      class="main-content"
    >
      <div>
        PhoneListPage
      </div>
    </section>
    <footer>
      Footer
    </footer>
  </div>
</div>
`;
